### PR TITLE
Update shellcheck url.

### PR DESCRIPTION
**Pull request by ShigekiKarita**
According to our [CI](https://github.com/espnet/espnet/runs/1137149385?check_suite_focus=true)

> Please update to the new URL:
https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz
For more information, see:
https://github.com/koalaman/shellcheck/issues/1871
PS: Sorry for breaking your build :(
You are downloading ShellCheck from an outdated URL!

I also found the second `tee` forgot `-a` (append). **So all `ci utils doc egs2/TEMPLATE/*/scripts egs2/TEMPLATE/*/setup.sh tools/*.sh`  shellchek were ignored!**
https://github.com/espnet/espnet/blob/4343538a82db99d511fdde1c6d85cdcef4069f08/ci/test_shell.sh#L22-L24

For new detected errors, see https://github.com/espnet/espnet/runs/1137265794?check_suite_focus=true#step:7:136

### CODE REVIEW

File: tools/setup_anaconda.sh [hunk 1]
```diff
@@ -57,6 +58,8 @@ fi
 conda install -y pip setuptools
 
 cat << EOF > activate_python.sh
+#!/bin/bash
+# THIS FILE IS GENERATED BY tools/setup_anaconda.sh
```
Comment (kamo-naoyuki) [inline tools/setup_anaconda.sh:62]: Would you also write these lines in tools/setup_venv.sh ?

File: tools/setup_python.sh [hunk 2]
```diff
@@ -37,6 +37,8 @@ fi
 # Change the user site packages dir from "~/.local"
 echo "Warning: Setting PYTHONUSERBASE"
 cat << EOF > activate_python.sh
+#!/bin/bash
+# THIS FILE IS GENERATED BY tools/setup_anaconda.sh
```
Comment (kamo-naoyuki) [inline tools/setup_python.sh:41]:
```suggestion
# THIS FILE IS GENERATED BY tools/setup_python.sh
```

File: tools/setup_anaconda.sh (hunk 1 above)
Comment (ShigekiKarita) [reply tools/setup_anaconda.sh:62]: Done

File: tools/setup_python.sh (hunk 2 above)
Comment (ShigekiKarita) [reply tools/setup_python.sh]: Done

**Review by kamo-naoyuki — APPROVED** [review]
LGTM

**Pull request merged**